src/main/java/com/bookstore/dto/item/CartItemRequestDto.java

```diff
@@ -0,0 +1,15 @@
+package com.bookstore.dto.item;
+
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+
+@Data
+public class CartItemRequestDto {
+    @NotNull
+    @Min(value = 1)
```

Comment:
```suggestion
    @Min(1)
```
`value` considered as default param, so it's not required to specify it if you only pass 1 parameter to the annotation